001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java

```diff
@@ -0,0 +1,56 @@
+package projectorangehrm;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class LogInPageOrangeHRM {
+
+	public static void main(String[] args) {
+		
+		//1.Launch Chrome Browser
+		WebDriver driver=new ChromeDriver();
+		
+		//maximize the browser(Method chaining)
+		driver.manage().window().maximize();
+		
+		//2. Navigate to OrangeHRM url
+		driver.get("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");
+		
+		//wait in Login Page
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(3));

```

Comment:
move this as variable into prop file again...... for ex : 3 is hardcoded !

Reply:
I made all the changes in 002-loginwithtestng